### DISCUSSION TITLE
Put in Masque Skins

### DISCUSSION THREAD

**Discussion by zowyck** [SFX-WoW/Masque#168] [Issues] [answered]
I try put Masque skins in same folder work in version 700 however in version 701 no work fine and i use en older version 6.2.1 last octuber go to play original

put addons in folder skins and remove all toc's

World of Warcraft\Interface\AddOns\Masque\Skins\Masque_Caith
World of Warcraft\Interface\AddOns\Masque\Skins\Masque_DaedUI
World of Warcraft\Interface\AddOns\Masque\Skins\Masque_Darkscail
World of Warcraft\Interface\AddOns\Masque\Skins\Masque_ElvUIesque
World of Warcraft\Interface\AddOns\Masque\Skins\Masque_Onyx
World of Warcraft\Interface\AddOns\Masque\Skins\Masque_Shining

and put this in toc MASQUE

Skins\Masque_DaedUI\Skin.lua
Skins\Masque_Darkscail\Skin.lua
Skins\Masque_Onyx\Skin.lua
Skins\Masque_Onyx\Skin2.lua
Skins\Masque_Shining\Skin.lua
Skins\Masque_Shining\Skin2.lua
Skins\Masque_Caith\Skin.lua
Skins\Masque_ElvUIesque\Skin.lua

work fine in version 700 and older however in version 701 no work today bartender hide and update from version 621 and see problem.

**Comment by StormFX** [comment]
Any skin that's not included with Masque is its own add-on. I can't really be bothered to fix a problem with improper installation. Install them separately, as they're supposed to be.

World of Warcraft\Interface\AddOns\Masque_Caith
World of Warcraft\Interface\AddOns\Masque_DaedUI
World of Warcraft\Interface\AddOns\Masque_Darkscail
World of Warcraft\Interface\AddOns\Masque_ElvUIesque
World of Warcraft\Interface\AddOns\Masque_Onyx
World of Warcraft\Interface\AddOns\Masque_Shining

With the ToCs intact.

**Comment by StormFX** [comment]
For the record, using the method you're using, you have to copy those files and edit the ToC EVERY TIME Masque is updated because updating REMOVES the entire folder and re-installs it.

**Comment by zowyck** [comment]
i am install this form to use more less list addons in game, in pandaria, and draenor use always this form for more lite list
and if I understand that each update have to manually update the toc not bother me, but I understand not see it as a bug, although in earlier versions and the 700 version works well.

**Comment by StormFX** [comment, marked as answer]
It's not a bug if you install them other than intended. Nothing was changed in 7.0.1 that would cause this. The issue is on your end.
